src/Etsy/methods.json

```diff
@@ -759,9 +759,9 @@
     "params": {
       "listing_id": "int",
       "products": "stringJSON",
-      "price_on_property": "array(int)",
-      "quantity_on_property": "array(int)",
-      "sku_on_property": "array(int)"
+      "price_on_property": "string",
```

Comment:
Take a look to `tags` param in `createListing`. It's not the same case because the definition says that is `array(string)` and in this case is `array(int)`, but Etsy for the `tags` case expects values like: `["tag1, tag2, tag3"]` and this library: 
```
{
...
"tags": array("tag1, tag2, tag3"),
...
}
```

Can you try something like this?
```
{
...
"price_on_property": array("1, 2, 3, 4"),
...
}
```